packages/vue-components/src/TipBox.vue

```diff
@@ -251,61 +257,29 @@ export default {
       return {};
     },
     getBootstrapAlertStyle() {
-      switch (this.type) {
-      case 'warning':
-        return 'alert-warning';
-      case 'info':
-        return 'alert-info';
-      case 'definition':
-        return 'alert-primary';
-      case 'success':
-      case 'tip':
-        return 'alert-success';
-      case 'important':
-      case 'wrong':
-        return 'alert-danger';
-      default:
-        return 'alert-default';
+      const { baseStyle, colorStyle } = classifyBootstrapStyle(this.type, this.theme);
```

Comment:
Would placing this in a `computed` be better since there are a few usages in other methods? This can avoid repeated calls to `classifyBootstrapStyle`.

Reply:
Ok this sounds like a good improvement as well, still quite new to Vue so this tip was really helpful!